src/jekyll/zeppelin.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: global
+displayTitle: Apache Zeppelin running with Spark on Kubernetes
+title: Apache Zeppelin running with Spark on Kubernetes
+description: User Documentation for Apache Zeppelin running with Spark on Kubernetes
+tag: experimental
+---
+
+**Apache Zeppelin with Spark on Kubernetes is `experimental`**
+
+> At the time being, the needed code is not integrated in the `master` branches of `apache-zeppelin` nor the `apache-spark-on-k8s/spark` repositories.
+> You are welcome to already ty it out and send any feedback and question.
+
+First things first, you have to choose the following modes in which you will run Zeppelin with Spark on Kubernetes:
+
++ The `Kubernetes modes`: Can be `in-cluster` (within a Pod) or `out-cluster` (from outside the Kubernetes cluster).
++ The `Spark deployment modes`: Can be `client` or `cluster`.
+
+Only three combinations of these options are supported:
+
+1. `in-cluster` with `spark-client` mode.
+2. `in-cluster` with `spark-cluster` mode.
+3. `out-cluster` with `spark-cluster` mode.
+
+For now, to be able to test these combinations, you need to build specific branches (see hereafter) or to use third-party Helm charts or Docker images. The needed branches and related PR are listed here:
```

Comment:
As discussed in the meeting today, we want to ensure that these branches merge before we can publish documentation. 

cc @felixcheung @erikerlandson @liyinan926 @mccheah